src/services/transcript-parser.js

```diff
@@ -827,37 +832,76 @@ export const getMatchedTranscriptLines = (searchHits, query, transcripts) => {
     }
     const matchOffset = mappedText.toLocaleLowerCase().indexOf(qStr);
     if (matchOffset !== -1 && transcirptId != undefined) {
```

Comment:
```suggestion
    if (matchOffset !== -1 && transcriptId != undefined) {
```

Not a blocker, but all mentions of `transcriptId` in this function are typoed. It is still all working because the typo is consistent, but would probably be good to clean up.